api/controllers/SessionsController.js

```diff
@@ -5,52 +5,39 @@
  * @help        :: See https://sailsjs.com/docs/concepts/actions
  */
 
-const fs = require('fs');
-const path = require('path');
 const conditionHelper = require('../utils/ConditionHelpers');
 
 module.exports = {
-  index: function (req, res) {
-
-    let directory = path.resolve(__dirname, '../../sessions');
+  index: async function (req, res) {
+
+    const reports = await MedicalReport.findAll({limit: 10, attributes: ['id', 'firstName', 'lastName', 'socialInsuranceNumber']});
+    sails.log(`Query returned: ${JSON.stringify(reports)}`);
+    const viewModel = reports.map(x =>  {
+      return {
+        id: x.id,
+        dateRecieved: 'Not Implemented',
+        applicant: `${x.lastName}, ${x.firstName}`,
+        sin: x.socialInsuranceNumber,
+        lastViewed: 'Not Implemented'
+      };
+    });
 
-    fs.readdir(directory, (err, files) => {
-      if (!err) {
-        _.pull(files, '.gitignore');
-        res.view('pages/sessions', {
-          files: files
-        });
-      }
+    const totalReports = await MedicalReport.count();
+    sails.log(`totalReports: ${totalReports}`);
+    return res.view('pages/sessions', {
+      reports: viewModel,
+      total: totalReports
     });
   },
 
-  download: function (req, res) {
-    const filename = req.params.session;
-    const sessionFile = path.resolve(__dirname, '../../sessions/' + filename);
-
-    var SkipperDisk = require('skipper-disk');
-    var fileAdapter = SkipperDisk(/* optional opts */);
-
-    // set the filename to the same file as the user uploaded
-    res.set('Content-disposition', 'attachment; filename=' + filename);
-
-    // Stream the file down
-    fileAdapter.read(sessionFile)
-      .on('error', (err) => {
-        return res.serverError(err);
-      })
-      .pipe(res);
-  },
+  view: async function (req, res) {
 
-  view: function (req, res) {
-    const filename = req.params.session;
-    const sessionFile = path.resolve(__dirname, '../../sessions/' + filename);
+    const medicalReport = await MedicalReport.findOne({ where: {id: req.params.session}});
+    sails.log(`medicalReport: ${JSON.stringify(medicalReport, 2)}`);
```

Comment:
can call `.toJSON()` on the model

Reply:
toJSON doesn't return a string it returns a json object so you need to JSON.stringify the output of toJSON